Makefile

```diff
@@ -2,7 +2,6 @@ SHELL := /bin/sh
 KVER ?= $(if $(KERNELRELEASE),$(KERNELRELEASE),$(shell uname -r))
 KSRC ?= $(if $(KERNEL_SRC),$(KERNEL_SRC),/lib/modules/$(KVER)/build)
 FIRMWAREDIR := /lib/firmware/rtw88
-PWD := $(shell pwd)
```

Comment:
This is not needed? I see something called PWD is still being used.

(I don't know make that well.)

Reply:
We can use $$PWD to get the current working directory from environment, so that line is not needed.